main/src/cgeo/geocaching/ui/ContinuousRangeSlider.java

```diff
@@ -64,6 +73,70 @@ private void init() {
+                    // still on the same thumb, and not having moved it out of grace area?
+                    if (lastThumb >= 0 && lastThumb < values.size() && Math.abs(lastValue - values.get(lastThumb)) < grace) {
+                        final String title = String.format(getContext().getString(R.string.number_input_title), String.format(Locale.getDefault(), "%.2f", slider.getValueFrom()), String.format(Locale.getDefault(), "%.2f", slider.getValueTo()));
+                        final String defaultValue = String.format(Locale.getDefault(), "%.2f", lastValue);
+                        final int inputType = InputType.TYPE_CLASS_NUMBER | InputType.TYPE_NUMBER_FLAG_DECIMAL;
+                        final Consumer<String> listener = input -> {
+                            float newValue;
+                            try {
+                                newValue = Float.parseFloat(input);
+                                if (newValue > slider.getValueTo()) {
+                                    newValue = slider.getValueTo();
+                                    Toast.makeText(getContext(), R.string.number_input_err_boundarymax, Toast.LENGTH_SHORT).show();
+                                }
+                                if (newValue < slider.getValueFrom()) {
+                                    newValue = slider.getValueFrom();
+                                    Toast.makeText(getContext(), R.string.number_input_err_boundarymin, Toast.LENGTH_SHORT).show();
+                                }
+                                if (lastThumb == 0) {
+                                    // we want to set lower bound, but new value is larger than current upper bound, so swap
+                                    if (newValue > values.get(1)) {
+                                        slider.setValues(values.get(1), newValue);
+                                    } else {
+                                        slider.setValues(newValue, values.get(1));
+                                    }
+                                } else {
+                                    // we want to set upper bound, but new value is smaller than current lower bound, so swap
+                                    if (newValue < values.get(0)) {
+                                        slider.setValues(newValue, values.get(0));
+                                    } else {
+                                        slider.setValues(values.get(0), newValue);
+                                    }
+                                }
+                            } catch (NumberFormatException e) {
+                                Toast.makeText(getContext(), R.string.number_input_err_format, Toast.LENGTH_SHORT).show();
+                            }
+                        };
+                        SimpleDialog.of((Activity) getContext()).setTitle(TextParam.text(title)).input(inputType, defaultValue, null, "", listener);
```

Comment:
You might want to use the SimpleDialog.ofContext method instead

Reply:
done